app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'bjohnson@codeforamerica.org'
```

Comment:
what's the status of getting this setup with adco?

Reply:
I don't think Jason responded, but I can remind him.